src/commands/createVirtualMachine/ConfirmPassphraseStep.ts

```diff
@@ -0,0 +1,36 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzureNameStep } from "vscode-azureextensionui";
+import { ext } from "../../extensionVariables";
+import { localize } from "../../localize";
+import { IVirtualMachineWizardContext } from "./IVirtualMachineWizardContext";
+
+export class ConfirmPassphraseStep extends AzureNameStep<IVirtualMachineWizardContext> {
```

Comment:
This should extend `AzureWizardPromptStep`, not `AzureNameStep` and remove the related name stuff